ctapipe/io/tableloader.py

```diff
@@ -175,37 +174,33 @@ class TableLoader(Component):
     input_url = traits.Path(
         directory_ok=False, exists=True, allow_none=True, default_value=None
     ).tag(config=True)
-
-    load_dl1_images = traits.Bool(False, help="load extracted images").tag(config=True)
-    load_dl1_parameters = traits.Bool(
-        True, help="load reconstructed image parameters"
-    ).tag(config=True)
-    load_dl1_muons = traits.Bool(False, help="load muon ring parameters").tag(
+    dl1_images = traits.Bool(False, help="load extracted images").tag(config=True)
+    dl1_parameters = traits.Bool(True, help="load reconstructed image parameters").tag(
         config=True
     )
+    dl1_muons = traits.Bool(True, help="load muon ring parameters").tag(config=True)
 
-    load_dl2 = traits.Bool(False, help="load available dl2 stereo parameters").tag(
+    dl2 = traits.Bool(True, help="load available dl2 stereo parameters").tag(
         config=True
     )
 
-    load_simulated = traits.Bool(False, help="load simulated shower information").tag(
+    simulated = traits.Bool(True, help="load simulated shower information").tag(
         config=True
     )
-    load_true_images = traits.Bool(False, help="load simulated shower images").tag(
+    true_images = traits.Bool(False, help="load simulated shower images").tag(
         config=True
     )
-    load_true_parameters = traits.Bool(
-        False, help="load image parameters obtained from true images"
+    true_parameters = traits.Bool(
+        True, help="load image parameters obtained from true images"
     ).tag(config=True)
 
-    load_instrument = traits.Bool(
-        False, help="join subarray instrument information to each event"
+    instrument = traits.Bool(
+        True, help="join subarray instrument information to each event"
     ).tag(config=True)
 
-    load_observation_info = traits.Bool(
-        False, help="join observation information to each event"
+    observation_info = traits.Bool(
+        True, help="join observation information to each event"
```

Comment:
also should be false by default I think. Keep the defaults to be just a minimal subset